x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -175,7 +199,48 @@ export const AlertsList: React.FunctionComponent = () => {
     }
   }
 
+  async function loadAlertsTotalStatuses() {
+    let alertsStatuses = {};
+    try {
+      AlertExecutionStatusValues.forEach(async (status: string) => {
+        const alertsTotalResponse = await loadAlerts({
+          http,
+          page: { index: 0, size: 0 },
+          searchText,
+          typesFilter,
+          actionTypesFilter,
+          alertStatusesFilter: [status],
+        });
+        setAlertsStatusesTotal({ ...alertsStatuses, [status]: alertsTotalResponse.total });
+        alertsStatuses = { ...alertsStatuses, [status]: alertsTotalResponse.total };
```

Comment:
**nit:** the usage of `alertsStatuses` is a bit hard to understand. Would it be possible to use a mix of `await Promise.all([...]);` and `AlertExecutionStatusValues.map(async (status: string) => { ... });` and then call `setAlertsStatusesTotal` once all the calls are finished?